src/main/java/uk/gov/hmcts/divorce/solicitor/event/SolicitorCreateApplication.java

```diff
@@ -91,6 +96,27 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
 
         final CaseDetails<CaseData, State> result = solicitorCreateApplicationService.aboutToSubmit(details);
 
+        MarriageDetails marriageDetails = result.getData().getApplication().getMarriageDetails();
+
+        Applicant applicant1 = result.getData().getApplicant1();
+        Applicant applicant2 = result.getData().getApplicant2();
+
+        if (OPPOSITE_SEX_COUPLE.equals(marriageDetails.getFormationType())) {
```

Comment:
this could probably be moved to a model somewhere, either:

application.getPartnerGender(applicant1.getGender())

or

applicant.getPartnerGender(application.getMarriageDetails().getFormationType())